src/components/apphost.js

```diff
@@ -70,6 +71,20 @@ function getDeviceProfile(item) {
             });
         }
 
+        if (preferredTranscodeAudio !== '') {
```

Comment:
I would move the variable closer to where it is used and remove the strict comparison.
```suggestion
        const preferredTranscodeAudio = appSettings.preferredTranscodeAudio();

        if (preferredTranscodeAudio) {
```